docker/build-tools/Dockerfile

```diff
@@ -49,13 +49,19 @@ ENV PROTOLOCK_VERSION=v0.14.0
 ENV PROTOTOOL_VERSION=v1.8.0
 ENV SHELLCHECK_VERSION=v0.7.0
 ENV UPX_VERSION=3.95
+ENV DOCKER_VERSION=18.06.1
```

Comment:
this is a super old version - I'd recommend using `19.03.1`

Reply:
Done